Week1/Section3/Exercise7/Source7.cpp

```diff
@@ -8,6 +8,28 @@
 
 #include <iostream>
 
+bool IsNumberRepeated(int ArrayToCheck[5], int Number)
+{
+	for (int i = 0 ; i < 5 ; i++)
+	{
+		if (ArrayToCheck[i] == Number)
+		{
+			std::cout << "Do not repeat numbers!" << std::endl;
+			return true;
+		}
+	}
+	return false;
+	/*
+	for (int x : ArrayToCheck)
```

Comment:
Commented-out code should be rare and documented. If there is no reason to have this code, you should delete it instead of commenting it.

Reply:
Sorry, my bad, it was a failed attempt that I forgot to delete :P